code-pushup.config.ts

```diff
@@ -0,0 +1,61 @@
+import eslintPlugin, { eslintConfigFromNxProjects } from '@code-pushup/eslint-plugin';
+import type { CoreConfig } from '@code-pushup/models';
+import 'dotenv/config';
+
+const config: CoreConfig = {
+  upload: {
+    server: 'https://portal-api-r6nh2xm7mq-ez.a.run.app/graphql',
+    apiKey: process.env.CP_API_KEY,
+    organization: 'push-based',
+    project: 'user-flow',
+  },
+  plugins: [
+    await eslintPlugin(await eslintConfigFromNxProjects()),
+  ],
+  categories: [
```

Comment:
You could use groups:

```ts
{
      slug: 'bug-prevention',
      title: 'Bug prevention',
      refs: [{ type: 'group', plugin: 'eslint', slug: 'problems', weight: 1 }],
    },
    {
      slug: 'code-style',
      title: 'Code style',
      refs: [
        { type: 'group', plugin: 'eslint', slug: 'suggestions', weight: 1 },
      ],
    },
```

Reply:
I don't know what you mean. 

What are groups. 

Actually how do these categories affect the results and what is tested. 

Where do i know what categories and groups do i have available ? 

Are their any docs on this?

Reply:
Our main documentation on this is the [glossary](https://github.com/code-pushup/cli/wiki/Glossary). Each plugin then has its own groups and audits documented in its documentation (e.g. [ESLint](https://github.com/code-pushup/cli/blob/main/packages/plugin-eslint/README.md)).

[Categories](https://github.com/code-pushup/cli/wiki/Glossary#category) are useful for grouping similar audits or groups together. They can then produce one score for the whole category. They are **defined by the user** in the configuration file. By defining weights to category `refs`, one can give more weight to one reference (audit or group) over another.

A [group](https://github.com/code-pushup/cli/wiki/Glossary#group) is a collection of audits within a given plugin which is **provided by the plugin** itself. It is helpful when there is a plugin with many audits (such as ESLint, where each rule is a separate audit) to have a shorter configuration file. One can then reference the whole group instead of the audits in a category.

The configuration you provided combines the manual and group approach (`prefer-const` or `no-var` is likely to be in one of the groups). What @BioPhoton copy pasted is an example of our configuration on Code PushUp CLI project.

If this is all too much, that is understandable. I am happy to clarify this or help you configure Code PushUp on a call with you.

Reply:
Yes, i will need a call to understand a properly configure it. Thanks a lot